Doc/reference/datamodel.rst

```diff
@@ -2811,7 +2811,7 @@ through the object's keys; for sequences, it should iterate through the values.
    the accepted keys should be integers and slice objects.  Note that the
    special interpretation of negative indexes (if the class wishes to emulate a
    :term:`sequence` type) is up to the :meth:`__getitem__` method. If *key* is
-   of an inappropriate type, :exc:`TypeError` may be raised; if of a value
+   of an inappropriate type, :exc:`TypeError` may be raised; if a value
```

Comment:
Alternative wordings:

```suggestion
   of an inappropriate type, :exc:`TypeError` may be raised; if *key* is a value
```

```suggestion
   of an inappropriate type, :exc:`TypeError` may be raised. If *key* is a value
```

I prefer either of these to the one I PR'd, but I wanted to make a PR with the smallest possible fix.